tensorflow/python/keras/metrics.py

```diff
@@ -2856,3 +2856,325 @@ def get(identifier):
   else:
     raise ValueError('Could not interpret '
                      'metric function identifier: %s' % identifier)
+
+class FBetaScore(Metric):
+    """Computes F-Beta Score.
+
+    This is the weighted harmonic mean of precision and recall.
+    Output range is [0, 1].
+
+    F-Beta = (1 + beta^2) * ((precision * recall) /
+                             ((beta^2 * precision) + recall))
+
+    `beta` parameter determines the weight given to the
+    precision and recall.
+
+    `beta < 1` gives more weight to the precision.
+    `beta > 1` gives more weight to the recall.
+    `beta == 1` gives equal weight to precision and recall.
+
+    Args:
+       num_classes : Number of unique classes in the dataset.
+       average : Type of averaging to be performed on data.
+                   Acceptable values are None, micro, macro and
+                   weighted.
```

Comment:
Are these strings? Could we use quotes around strings for all the usages to make it clear?